com.microsoft.mrtk.input/Simulation/Devices/SimulatedController.cs

```diff
@@ -29,6 +29,11 @@ namespace Microsoft.MixedReality.Toolkit.Input.Simulation
         Preserve]
     internal class MRTKSimulatedController : XRSimulatedController
     {
+        /// <summary>
+        /// The current simulation mode for this controller;
+        /// </summary>
+        public ControllerSimulationMode SimulationMode;
```

Comment:
code style: can this be a property instead of a public field?

Reply:
Done